workbench-core/accounts/src/services/projectService.ts

```diff
@@ -80,4 +71,220 @@ export default class ProjectService {
 
     return Promise.resolve({ data: projectsResponse.Items as unknown as Project[] });
   }
+
+  /**
+   * Creates a new project
+   *
+   * @param params - the required fields to create a new project
+   * @param user - authenticated user creating the project
+   * @returns Project object of new project
+   */
+  public async createProject(params: CreateProjectRequest, user: AuthenticatedUser): Promise<Project> {
+    // Verify caller is an IT Admin--TODO implement after dynamic AuthZ
+    // const userGroupsForCurrentUser: string[] = await this._dynamicAuthorizationService.getUserGroups(user.id);
+    // if(userGroupsForCurrentUser.length !== 1 || userGroupsForCurrentUser[0] !== 'ITAdmin') {
+    //   throw Boom.forbidden('Only IT Admins are allowed to create new Projects.');
+    // }
+
+    // Verify project name is unique and cost center exists
+    const resultsFromValidityChecks = await Promise.all([
```

Comment:
Not blocking, but it might actually be worth doing these one at a time so we can easily control where the failure logic is for each validity check.

Reply:
Kevin wanted them done in parallel.